diesel/src/pg/types/network_address.rs

```diff
@@ -0,0 +1,194 @@
+extern crate ipnetwork;
+
+use std::io::prelude::*;
+use std::error::Error;
+use std::net::{Ipv4Addr, Ipv6Addr};
+
+use pg::Pg;
+use types::{self, ToSql, IsNull, FromSql};
+use self::ipnetwork::{IpNetwork,Ipv4Network,Ipv6Network};
+
+// const PGSQL_AF_INET: u8 = libc::AF_INET + 0;
+// const PGSQL_AF_INET6: u8 = libc::AF_INET + 1;
+const PGSQL_AF_INET: u8 = 2 + 0;
+const PGSQL_AF_INET6: u8 = 2 + 1;
```

Comment:
To tell the truth, I'm not confident. To err on the side of caution I want to use libc crate. Is it possible to depend on libc only for using a constant?